tornado/ioloop.py

```diff
@@ -43,8 +43,9 @@
 import time
 import traceback
 import math
+from multiprocessing import cpu_count
```

Comment:
`tornado.process` has a more robust wrapper around `cpu_count`, although I'm not sure if it's needed any more (it mainly dates back to when we supported python 2.5).